osmomath/exp2_test.go

```diff
@@ -0,0 +1,296 @@
+package osmomath_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/osmosis-labs/osmosis/v13/app/apptesting/osmoassert"
+	"github.com/osmosis-labs/osmosis/v13/osmomath"
+	"github.com/osmosis-labs/osmosis/v13/osmoutils"
+)
+
+var (
+	// minDecTolerance minimum tolerance for sdk.Dec, given its precision of 18.
+	minDecTolerance = sdk.MustNewDecFromStr("0.000000000000000001")
+)
+
+func TestExp2ChebyshevRationalApprox(t *testing.T) {
+	smallValue := osmomath.MustNewDecFromStr("0.00001")
+	smallerValue := osmomath.MustNewDecFromStr("0.00000000000000000001")
```

Comment:
What is this smaller than & what is relation to minDecTolerance (sorry not sure if theres some important relation)

Reply:
I used these to implement test cases closer to the boundaries.

The reason is that the errors tend to be large at the boundaries for non-Chebyshev approximations. This is just to make sure that, as expected, Chebyshev does not allow for larger errors close to boundaries. I will add a comment relaying this